### DISCUSSION TITLE
Low power consumption

### DISCUSSION THREAD

**Discussion by SigmaPic** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#351] [Ideas]
Hello

@visualapproach Did you tried to play with the setSleepMode () API:
[https://arduino-esp8266.readthedocs.io/en/latest/esp8266wifi/generic-class.html?highlight=modem#wifi-power-management-dtim](https://arduino-esp8266.readthedocs.io/en/latest/esp8266wifi/generic-class.html?highlight=modem#wifi-power-management-dtim)

It allow to configure the modem sleep mode that put the Wifi in sleep mode but keep the connection on.
It means that it is transparent for the user.
I just allow to save power supply.

I tried and it works : 
![image](https://user-images.githubusercontent.com/2431226/180499920-fa3214eb-e26e-48f2-b383-c4e16872e6fa.png)

I didn't check the consuption before and after. I will maybe check that next week.

It's interesting because it allow to reduce the current from the power supply that is maybe not designed to drain the current consumption from the ESP.

**Comment by visualapproach** [comment]
@SigmaPic I have not played with that. Let me know how it turns out.
